tests/rt/devel_packages.pm

```diff
@@ -0,0 +1,26 @@
+# SUSE's openQA tests
+#
+# Copyright Â© 2009-2013 Bernhard M. Wiedemann
+# Copyright Â© 2012-2016 SUSE LLC
```

Comment:
check your file encodings. Also for new files the copyright should be correct, i.e. only 2016 (maybe 2015 or earlier, depends how long you had this lying around ;-) )